pom.xml

```diff
@@ -15,9 +15,9 @@
         <module>helpick-service</module>
     </modules>
     <properties>
-        <java.version>11</java.version>
-        <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.target>11</maven.compiler.target>
+        <java.version>8</java.version>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
```

Comment:
dlaczego zmieniamy wersje Java?

Reply:
Mój błąd, nie poprawiłem po tym, jak nie chciał się consoleApp odpalić. Zejście do 8 nic by nam nie robiło, bo nic z wyższych wersji nie mamy, ale to tak na marginesie. Przywróciłem do 11.